spec/rubocop/ast/str_node_spec.rb

```diff
@@ -56,4 +56,31 @@
       it { expect(str_node.heredoc?).to be(true) }
     end
   end
+
+  describe '#value' do
+    context 'with a normal string' do
+      let(:source) { "'foo'" }
+
+      it { expect(str_node.value).to eq('foo') }
+    end
+
+    context 'with a string with interpolation' do
+      let(:source) { '"foo #{bar} baz"' }
+
+      it { expect(str_node.value).to eq('foo  baz') }
```

Comment:
This may or may not be the desired behaviour, but in any case it is the current one. If we want to change this we can do so down the line. At least now the behaviour is documented. 🙂